backend/app/Http/Controllers/JourneyController.php

```diff
@@ -40,7 +40,7 @@ public function store(Request $request): JsonResponse
         );
 
         $journey = Journey::create($validated);
-        $journey->users()->attach(auth()->id());
+        $journey->users()->attach(auth()->id(), ['role' => 1]);
```

Comment:
Maybe update PHPDoc to say what resource is created and also explain why role is set to 1?